### PR TITLE
Cheminfo generator

### PR DESCRIPTION
**Pull request by maasencioh**
Now the generator has the cheminfo option and ask for the organization if it's not given

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,7 @@ This will:
 
 ### generate
 
-Generates a basic project structure based in the giving organization (see [generator-mljs-packages](https://github.com/mljs/generator-mljs-packages)).
-You need to run the command from the root of the project's directory.
+Generates a basic project structure based in the giving organization. You need to run the command from the root of the project's directory (previously you had to create it and clone it).
```
Comment (targos) [inline README.md:50]: `based on`

**Comment by targos** [comment]
LGTM

**Comment by targos** [comment]
Thanks! merged and released as 1.5.0

File: README.md (hunk 1 above)
Comment (maasencioh) [inline README.md:50]: I have a serious problem with that jajajajajajajaja sorry

**Pull request closed without merging**